lib/browser/api/web-contents.js

```diff
@@ -511,11 +511,13 @@ WebContents.prototype._init = function () {
     // Make new windows requested by links behave like "window.open".
     this.on('-new-window', (event, url, frameName, disposition,
       rawFeatures, referrer, postData) => {
-      const { options, additionalFeatures } = convertFeaturesString(rawFeatures, frameName);
+      const { options, webPreferences, additionalFeatures } = parseFeatures(rawFeatures);
       const mergedOptions = {
         show: true,
         width: 800,
         height: 600,
+        title: frameName,
+        webPreferences,
         ...options
```

Comment:
The code was previously careful to not let a `webPreferences=...` string in the feature string override the `webPreferences` that's built from other things. having `...options` come after `webPreferences` in this list makes me wonder if it's possible for `options` to contain a `webPreferences` key that could cause issues?